src/Rector/Class_/CreateOrUpdateFormDataRector.php

```diff
@@ -25,8 +25,11 @@ final class CreateOrUpdateFormDataRector extends AbstractRector implements Confi
 
     public const FORM_DATA_CLASS_TRAITS = 'form_data_class_traits';
 
-    private ?string $formDataClassParent = '\Nette\Utils\ArrayHash';
+    private string $formDataClassParent = '\Nette\Utils\ArrayHash';
```

Comment:
These should be without extra bracket like any other FQN. What would break without it?

```suggestion
    private string $formDataClassParent = 'Nette\Utils\ArrayHash';
```

Reply:
Nothing, I've already add it to https://github.com/rectorphp/rector-src/blob/main/src/NodeFactory/ClassWithPublicPropertiesFactory.php#L68 :) so I can remove it here